helm/application-namespace-prep/tests/authenticator-rolebinding_test.yaml

```diff
@@ -0,0 +1,66 @@
+suite: test authenticator-rolebinding
+
+templates:
+  - authenticator-rolebinding.yaml
+
+defaults: &defaultRequired
+  authnK8s.goldenConfigMap: authn-k8s-configmap
+  authnK8s.namespace: golden
+  
+
+tests:
+  #=======================================================================
+  - it: should not create a ConfigMap if ConfigMap creation is disabled
+  #=======================================================================
+    set:
+      authnRoleBinding.create: false
+    asserts:
+      - hasDocuments:
+          count: 0
+
+  #=======================================================================
+  - it: should use default values when those values are not set explicitly
+  #=======================================================================
+    set:
+      # Set required values
+      <<: *defaultRequired
+      test.mock.enable: true
+
```

Comment:
The line spacing before the `set:` sections, and between `set:` sections and `asserts:` is inconsistent in this test file (same for file that follows). I don't have a preference as long as it's mostly consistent.